packages/plugin-devtools/assets/DevTools.xml

```diff
@@ -122,6 +122,7 @@
             <t t-if="props.vNode.char">:
                 "<t t-esc="props.vNode.char"/>"
             </t>
+            <button class="logger" t-on-click="logVNode(props.vNode)">&gt;_</button>
```

Comment:
@aab-odoo Didn't you tell me that it was possible to use `console.log` in owl templates ?

Reply:
@dmo-odoo It's not. It will produce `this.console.log('blabla');`